api/src/UserManager/LoginRegister/Test/fetchInput.test.ts

```diff
@@ -0,0 +1,72 @@
+import {expect} from "@jest/globals";
+import * as bcrypt from 'bcrypt';
+
+export interface ILoginForm {
+    email: string,
+    password?: string,
+    passwordHash?: string,
+    age?: number,
+}
```

Comment:
This should be a DTO not an Interface, I think we need to think and establish where we will have interfaces.

for me Interfaces should exist for Services only

Reply:
The input Can be an interface the output at the end is différent in terms of keys

Perhaps a class is interesting for that but i think interface Can Do the job, idk